### PR TITLE
[docker] link embedded certs to /etc/ssl to avoid seting the envvar

### PR DESCRIPTION
**Pull request by xvello**
### What does this PR do?

The `SSL_CERT_DIR` does not seem that robust, this PR links the embedded cacert in the standard `/etc/ssl` location. Added `CURL_CA_BUNDLE` envvar so `curl` works with them too.

### CODE REVIEW

File: Dockerfiles/agent/Dockerfile [hunk 1]
```diff
@@ -10,12 +10,16 @@ WORKDIR /output
...
+ && echo "Cleanup and trimming" \
  && rm -rf usr etc/init lib \
     opt/datadog-agent/sources \
     opt/datadog-agent/embedded/share/doc \
     opt/datadog-agent/embedded/share/man \
  && find opt/datadog-agent/ -iname "*.a" -delete \
- && if [ -z "$WITH_JMX" ]; then rm -rf opt/datadog-agent/bin/agent/dist/jmx; fi
+ && if [ -z "$WITH_JMX" ]; then rm -rf opt/datadog-agent/bin/agent/dist/jmx; fi \
+ && echo "Linking embedded ssl certificates" \
+ && mkdir -p etc \
```
Comment (mfpierre) [inline Dockerfiles/agent/Dockerfile]: Not sure about the mkdir etc ? (and the `-p` option could be removed)

Comment (xvello) [reply Dockerfiles/agent/Dockerfile]: `etc` should indeed be in the package, but if it's not the `ln` invocation will fail. The `-p` makes `mkdir` ignore the error if it already exists. As `mkdir` has no other "ignore errors" flag, the alternative is to write a shell if condition.

Comment (mfpierre) [reply Dockerfiles/agent/Dockerfile]: Don't we want it to fail if we have no `etc`? 🤔

Comment (xvello) [reply Dockerfiles/agent/Dockerfile]: you have a point, `/etc/datadog-agent` should be present anyway ;)

**Review by mfpierre — APPROVED** [review]
🔐

**Pull request merged**